gaia-fast-list.js

```diff
@@ -443,6 +521,30 @@ Internal.prototype = {
     });
   },
 
+  getCacheKey: function() {
+    return `${this.el.tagName}:${this.el.id}:${location}`;
+  },
+
+  updateCache: function() {
+    if (!this.el.caching) return;
```

Comment:
`this.el` refers to the pubic `<gaia-fast-list>` element. `this.el.caching` refers to the public property/attribute on this node.